src/kubernetes/deployment.template.yaml

```diff
@@ -0,0 +1,161 @@
+              value: 'https://preprod-idamapi.reform.hmcts.net:3511'
+            - name: IDAM_S2S_AUTH_URL
+              value: 'http://rpe-service-auth-provider-aat.service.core-compute-aat.internal'
+            - name: IDAM_S2S_AUTH_TOTP_SECRET
+              value: "${CLAIM_STORE_S2S_SECRET}"
+            - name: IDAM_CASEWORKER_ANONYMOUS_USERNAME
+              value: "${ANONYMOUS_CASEWORKER_USERNAME}"
+            - name: IDAM_CASEWORKER_ANONYMOUS_PASSWORD
+              value: "${ANONYMOUS_CASEWORKER_PASSWORD}"
+            - name: IDAM_CASEWORKER_SYSTEM_USERNAME
+              value: "${SYSTEM_UPDATE_USERNAME}"
+            - name: IDAM_CASEWORKER_SYSTEM_PASSWORD
+              value: "${SYSTEM_UPDATE_PASSWORD}"
+            - name: OAUTH2_CLIENT_SECRET
+              value: "${OAUTH2_CLIENT_SECRET}"
+            - name: GOV_NOTIFY_API_KEY
+              value: "${NOTIFY_API_KEY}"
+            - name: FRONTEND_BASE_URL
+              value: 'https://moneyclaims.aat.platform.hmcts.net'
+            - name: RESPOND_TO_CLAIM_URL
+              value: 'https://moneyclaims.aat.platform.hmcts.net/first-contact/start'
+            - name: PDF_SERVICE_URL
+              value: 'http://cmc-pdf-service-aat.service.core-compute-aat.internal'
+            - name: DOCUMENT_MANAGEMENT_URL
+              value: 'false'
+            - name: CORE_CASE_DATA_API_URL
+              value: 'false'
+            - name: SEND_LETTER_URL
+              value: 'http://rpe-send-letter-service-aat.service.core-compute-aat.internal'
+            - name: SPRING_MAIL_HOST
+              value: 'mta.reform.hmcts.net'
+            - name: SPRING_MAIL_PORT
+              value: '25'
+            - name: SPRING_MAIL_TEST_CONNECTION
+              value: 'false'
+            - name: SPRING_MAIL_PROPERTIES_MAIL_SMTP_STARTTLS_ENABLE
+              value: 'true'
+            - name: SPRING_MAIL_PROPERTIES_MAIL_SMTP_SSL_TRUST
+              value: 'mta.reform.hmcts.net'
+            - name: STAFF_NOTIFICATIONS_SENDER
+              value: 'noreply@reform.hmcts.net'
+            - name: STAFF_NOTIFICATIONS_RECIPIENT
+              value: "${STAFF_EMAIL}"
+            - name: RPA_NOTIFICATIONS_SENDER
+              value: 'noreply@reform.hmcts.net'
+            - name: RPA_NOTIFICATIONS_SEALEDCLAIMRECIPIENT
+              value: "${STAFF_EMAIL}"
+            - name: RPA_NOTIFICATIONS_MORETIMEREQUESTEDRECIPIENT
+              value: "${STAFF_EMAIL}"
+            - name: RPA_NOTIFICATIONS_RESPONSERECIPIENT
+              value: "${STAFF_EMAIL}"
+            - name: RPA_NOTIFICATIONS_COUNTYCOURTJUDGEMENTRECIPIENT
+              value: "${STAFF_EMAIL}"
+            - name: CLAIM_STORE_TEST_SUPPORT_ENABLED
+              value: 'true'
+            - name: FEATURE_TOGGLES_EMAILTOSTAFF
+              value: 'true'
+            - name: ROOT_APPENDER
+              value: 'CMC'
+            - name: APPINSIGHTS_INSTRUMENTATIONKEY
+              value: 'TODO'
+
```

Comment:
You could add a configmap to this file and reference all variables from that, in preparation for https://tools.hmcts.net/jira/browse/CNP-758 if we go down that route

Reply:
happy to do it when the library supports it

Reply:
if it's part of this same file it'll just work.  See here for example: https://github.com/hmcts/jenkins-container-config/blob/65aac84384bd3e4b45f1027349b682b3f3def387/aks/jenkins.yaml#L31

Reply:
can move it to another file once the library is updated to use different files per env